packages/block-editor/src/store/reducer.js

```diff
@@ -1714,9 +1714,12 @@ export function automaticChangeStatus( state, action ) {
 		case 'SET_BLOCK_VISIBILITY':
 		case 'START_TYPING':
 		case 'STOP_TYPING':
+		case 'UPDATE_BLOCK_LIST_SETTINGS':
 			return state;
 	}
 
+	// TODO: This is a source of bug, as each time there's a change in timing,
```

Comment:
I think we need to add more info here about what the bug actually is, for future reference. Maybe the link to the PR? 🤔

Reply:
There's no identified bug, it's just a bad pattern (returning undefined from a reducer) that can cause bugs quickly and these bugs are very hard to track.